<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncluirDVD</class>
 <widget class="QDialog" name="IncluirDVD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Incluir DVD</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::MediaOptical"/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Titulo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>107</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>INCLUIR NOVO DVD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Nome">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Preco">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Preço:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Duracao">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Duração:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Nome_value">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>190</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Preco_value">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>190</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Especifico_value">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>190</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IncluirDVD</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IncluirDVD</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
